packages/react-native/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -21,32 +25,48 @@ export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
     }
 
     private _lastBreadcrumbId: number = TimeHelper.toTimestampInSec(TimeHelper.now());
-    private readonly _writer: AlternatingFileWriter;
+    private readonly _dest: WritableStream;
+    private readonly _writer: WritableStreamDefaultWriter;
+    private readonly _sink: FileChunkSink;
 
     constructor(
+        session: SessionFiles,
         private readonly _fileSystem: FileSystem,
-        private readonly _mainFile: string,
-        private readonly _fallbackFile: string,
         maximumBreadcrumbs: number,
     ) {
-        this._writer = new AlternatingFileWriter(
-            _mainFile,
-            _fallbackFile,
-            Math.floor(maximumBreadcrumbs / 2),
-            _fileSystem,
+        this._sink = new FileChunkSink({
+            maxFiles: 2,
+            fs: this._fileSystem,
+            file: (n) => session.getFileName(FileBreadcrumbsStorage.getFileName(n)),
+        });
+
+        this._dest = new WritableStream(
```

Comment:
what is dest and what is the difference between dest and writer? Maybe changing the variable name makes sense?